pkg/ipam/crd_eni.go

```diff
@@ -200,5 +205,20 @@ func configureENINetlinkDevice(link netlink.Link, cfg eniDeviceConfig) error {
 		return fmt.Errorf("failed to delete default route %q on link %q: %w", cfg.ip, link.Attrs().Name, err)
 	}
 
+	if option.Config.EnableIPSec {
+		interfaces := []string{cfg.name}
+		ctx, cancel := context.WithTimeout(context.TODO(), defaults.ExecTimeout)
```

Comment:
nit:
```suggestion
		ctx, cancel := context.WithTimeout(context.Background(), defaults.ExecTimeout)
```

`context.TODO()` and `context.Background()` are functionally absolutely equivalent, but given we simply want to create a timed context here, using Background seems reasonable to me.